desktop-widgets/tab-widgets/maintab.cpp

```diff
@@ -426,12 +426,12 @@ void MainTab::updateDiveInfo(bool clear)
 	ui.notes->setText(QString());
 	if (!clear) {
 		QString tmp(displayed_dive.notes);
-		if (tmp.indexOf("<table") != -1)
+		if (tmp.indexOf("<table") != -1) {
+			tmp.replace(QString("\n"), QString("<br>"));
 			ui.notes->setHtml(tmp);
-		else
+		} else
```

Comment:
if the `if` branch is wrapped with `{ }` then the `else` branch should be wrapped with `{ }` even if it has a single line.